llms/mlx_lm/generate.py

```diff
@@ -107,6 +107,26 @@ def setup_arg_parser():
         default=None,
         help="A file containing saved KV caches to avoid recomputing them",
     )
+    parser.add_argument(
+        "--quantized-kv-start",
+        help="Use a quantized KV cache from this step onwards.",
+        type=int,
+        default=None,
+    )
+    parser.add_argument(
+        "--kv-group-size",
+        type=int,
+        help="Group size for kv cache quantization. "
+        "--quantized-kv-start must be provided to have an effect.",
```

Comment:
Why require that param? Is `0` not a reasonable default?

Reply:
I changed this around so that `--kv-bits` determines whether we do KV cache quantization or not since I think it's a bit more intuitive that way.

I have the default `quantized-kv-start=5000` since that's the best performance trade off at the moment.